Mathlib/Topology/Basic.lean

```diff
@@ -1012,6 +1012,12 @@ theorem acc_principal_iff_cluster (x : X) (C : Set X) :
     AccPt x (𝓟 C) ↔ ClusterPt x (𝓟 (C \ {x})) := by
   rw [acc_iff_cluster, inf_principal, inter_comm, diff_eq]
 
+theorem clusterPt_principal {X : Type*} [TopologicalSpace X] {x : X} {C : Set X}
```

Comment:
Note that `h` is equivalent to `x ∈ closure C` by `mem_closure_iff_clusterPt`. Also, isn't this an iff?

Reply:
I can make it into an iff but will the other direction ever be used? We have both `Or` cases as theorems already.

Reply:
Well, the other thing nice about iffs is that you can use them in `rw`. That seems at least somewhat plausible.